realm/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -281,6 +281,10 @@ public Builder name(String filename) {
 
         /**
          * Sets the 64 bit key used to encrypt and decrypt the Realm file.
+         * <p>
+         * Note that some devices do not support the encryption used by Realm. These devices will instead throw a
```

Comment:
"Note that some devices..." -> "Note that a few older devices.." 
Which ones have we identified? I think we should list them (somewhere - maybe not here) so people can decide if they want to support them.